extra-scientific/boinc/autobuild/overrides/usr/share/applications/boinc.desktop

```diff
@@ -1,19 +1,22 @@
 [Desktop Entry]
 Type=Application
 Version=1.0
-Exec=/usr/bin/boincmgr
+Exec=/usr/bin/boincmgr -d /var/lib/boinc
 Path=/var/lib/boinc
```

Comment:
That PATH was insufficient to specify the data dir for the boincmgr. I still keep opening disconnected boincmgr without this fix. Tested on KDE5